src/transports/ipc/aipc.c

```diff
@@ -89,6 +90,14 @@ void nn_aipc_start (struct nn_aipc *self, struct nn_usock *listener)
     self->listener_owner.fsm = &self->fsm;
     nn_usock_swap_owner (listener, &self->listener_owner);
 
+#if defined NN_HAVE_WINDOWS
+    /* Get/Set security attribute pointer*/
+    nn_epbase_getopt (self->epbase, NN_SOL_SOCKET, NN_SEC_ATTR, &self->usock.sec_attr, &sz);
```

Comment:
These options should be defined in ipc.h.  (I'm less sure about the buffer options, but I think so, there as well.)  The handling  here in aipc.c is the right place.